geocoding_platform_interface/lib/src/models/location.dart

```diff
@@ -39,12 +39,12 @@ class Location {
       latitude.hashCode ^ longitude.hashCode ^ timestamp.hashCode;
 
   /// Converts a list of [Map] instances to a list of [Location] instances.
-  static List<Location> fromMaps(dynamic message) {
+  static List<Location>? fromMaps(dynamic message) {
```

Comment:
Here I would never expect a `null` value. Maybe an empty list if converting the `message` to a `Location` doesn't work.

Reply:
This is similar to my own comments. As a consumer of the library I would not want to handle this and an empty list would be fine.